README.md

```diff
@@ -0,0 +1,28 @@
+# LSP-copilot
+
+Github Copilot support for Sublime's LSP plugin.
+
+Uses [Copilot][] Distribution which uses OpenAI Codex to suggest code and entire functions in real-time right from your editor.
+
+### Prerequisites
+
+* An active subscription is required to Github Copilot.
+
+### Installation
+
+* Install [LSP][] and `LSP-copilot` from Package Control.
```

Comment:
> `LSP-copilot` from Package Control.

oh really 🙃 ?

Reply:
Hopefully soon! Probably should update to fit until it’s on PC